src/Dataverse/MSBuildTasks/Props/SolutionFileItems.props

```diff
@@ -14,7 +14,7 @@
         <EntityRelationships Include="$(DataverseSolutionSourceFolderFullPath)\Other\Relationships\*.xml"/>
         <OptionSets Include="$(DataverseSolutionSourceFolderFullPath)\OptionSets\**\*.xml"/>
         <SavedQueries Include="$(DataverseSolutionSourceFolderFullPath)\Entities\*\SavedQueries\*.xml" />
-        <WorkflowMetadata Include="$(DataverseSolutionSourceFolderFullPath)\Workflows\*.xaml"/>
+        <WorkflowMetadata Include="$(DataverseSolutionSourceFolderFullPath)\Workflows\**\*.xaml"/>
```

Comment:
Is this nesting supported by the solution packager?

Reply:
Removed